Assets/Scripts/Module/Sound/AudioConfigModel.cs

```diff
@@ -0,0 +1,49 @@
+using Agate.MVC.Base;
+using UnityEngine;
+
+namespace TankU.Sound
+{
+    [System.Serializable]
+    public class AudioConfigModel : BaseModel
+    {
+        private const string _prefsKey = "_Audios_";
+        
+        [SerializeField] private float _bgmVolume = 1f;
+        [SerializeField] private float _sfxVolume = 1f;
+
+        public float BgmVolume => _bgmVolume;
+        public float SfxVolume => _sfxVolume;
```

Comment:
ini kenapah dipisah tulung penjelasannya.

Reply:
not entirely sure tbh, tapi utk bisa ngesave settingsnya ke playerpref